konig-schemagen/src/main/java/io/konig/schemagen/gcp/SpannerTable.java

```diff
@@ -0,0 +1,120 @@
+package io.konig.schemagen.gcp;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openrdf.model.URI;
+
+import io.konig.activity.Activity;
+import io.konig.gcp.datasource.SpannerTableReference;
+import io.konig.shacl.Shape;
+
+public class SpannerTable {
+
+	private String tableName;
+	private Shape tableShape;
+	private URI tableClass;
+	
+	private transient Activity wasGeneratedBy;
+	private transient List<Field> fields = new ArrayList<>();
+	private SpannerTableReference reference; 
+	
+	class Field {
+
+		private String fieldName;
+		private FieldMode fieldMode;

```

Comment:
I'm not so sure it makes sense to reuse FieldMode (from BigQuery tables) in a Spanner table because I don't think REPEATED is a valid mode for a Spanner field.  Is that true?